src/components/AttachmentPicker/index.native.js

```diff
@@ -239,6 +240,7 @@ function AttachmentPicker({type, children}) {
     const pickAttachment = useCallback(
         (attachments = []) => {
             if (attachments.length === 0) {
+                onCancel.current();
```

Comment:
Let's make these names consistent.
I see `onCanceled` on web, `onCancel` on native.
Also, isn't there case where ` onCancel.current` is undefined?